packages/web/src/components/CategoryResults.tsx

```diff
@@ -44,7 +45,9 @@ const CategoryResults = ({
   const status = subcategoryStub
     ? subcategoryResourcesStatus
     : categoryResourcesStatus;
-  const resources = subcategoryStub ? subcategoryResources : categoryResources;
+  const resources = ResourceSchedule.sortByOpen(
```

Comment:
I'm not sure that this utility belongs as a method on the `ResourceSchedule` class since most methods . Thoughts on just having a generic `sortByOpen` utility instead?

Reply:
Totes!